src/gc/config.d

```diff
@@ -26,7 +26,7 @@ struct Config
 {
     bool disable;            // start disabled
     ubyte profile;           // enable profiling with summary when terminating program
-    string gc = "conservative"; // select gc implementation conservative|manual
+    const(char)[] gc = "conservative"; // select gc implementation conservative|manual
```

Comment:
`parse` in `gc.config.initialize` receives a string, doesn't that allow to keep the options as strings aswell? It would be surprising if the referred string would be changed after initialization...